### PR TITLE
fix: differentiate between query and path params in getSchemas

### PR DESCRIPTION
**Pull request by xPhentom**
While debugging, I noticed there was no difference being made between query params and path params.
So for my test with the petStore.yml I was getting true on `schemas.pathParams` even though there were no path params. It returned true because it found parameters in the query.
I'll write a test still but I just want to make sure if you also think this is indeed a bug :)

### CODE REVIEW

File: packages/swagger/src/generators/OperationGenerator.ts [hunk 1]
```diff
@@ -3,11 +3,12 @@ import { pascalCase } from 'change-case'
 import type { FileManager, File } from '@kubb/core'
 import { Generator } from '@kubb/core'
 
+import { isReference } from '../utils/isReference'
```
Comment (wouterroozeleer) [inline packages/swagger/src/generators/OperationGenerator.ts:6]: Did not mean to place this import somewhere else, must have been the prettier/eslint setup

File: packages/swagger/src/generators/OperationGenerator.ts [hunk 2]
```diff
@@ -53,14 +54,22 @@ export abstract class OperationGenerator<TOptions extends { oas: Oas } = { oas:
   }
 
   getSchemas(operation: Operation): OperationSchemas {
+    const parameters = operation.getParameters()
+
+    const queryParams = parameters.filter((param) => param.in === 'query')
```
Comment (wouterroozeleer) [inline packages/swagger/src/generators/OperationGenerator.ts]: param.in could be anything but in swagger documentation it feels like these are the main/only ones?
https://swagger.io/docs/specification/describing-parameters/

**Comment by stijnvanhulle** [comment]
@all-contributors please add @ xPhentom for getSchemas fix

**Comment by stijnvanhulle** [comment]
https://github.com/all-contributors please add @xPhentom for getSchemas fix

**Comment by stijnvanhulle** [comment]
@all-contributors  please add @xPhentom for getSchemas fix

**Comment by stijnvanhulle** [comment]
@all-contributors  please add @xPhentom for code

**Comment by stijnvanhulle** [comment]
@all-contributors  please add @stijnvanhulle  for code

**Review by stijnvanhulle — APPROVED** [review]

**Pull request merged**